mkt/websites/forms.py

```diff
@@ -0,0 +1,88 @@
+import happyforms
+from tower import ugettext as _
+from tower import ugettext_lazy as _lazy
+from tower import ungettext as ngettext
+
+from django import forms
+
+import mkt
+from mkt.constants.categories import CATEGORY_CHOICES
+from mkt.constants.regions import REGIONS_CHOICES_NAME
+from mkt.tags.utils import clean_tags
+from mkt.translations.fields import TransField
+from mkt.translations.widgets import TransInput, TransTextarea
+from mkt.websites.models import Website
+from mkt.tags.models import Tag
+from mkt.site.utils import slugify
+
+
+class WebsiteForm(happyforms.ModelForm):
+    categories = forms.MultipleChoiceField(
+        label=_lazy(u'Categories'), choices=CATEGORY_CHOICES,
+        widget=forms.CheckboxSelectMultiple)
+    description = TransField(label=_lazy(u'Description'),
+                             widget=TransTextarea(attrs={'rows': 4}))
+    keywords = forms.CharField(label=_lazy(u'Keywords'), required=False,
+                               widget=forms.Textarea(attrs={'rows': 2}))
+    name = TransField(label=_lazy(u'Name'), widget=TransInput())
+    preferred_regions = forms.MultipleChoiceField(
+        label=_lazy(u'Preferred Regions'), choices=REGIONS_CHOICES_NAME,
+        required=False, widget=forms.SelectMultiple(attrs={'size': 10}))
+    short_name = TransField(label=_lazy(u'Short Name'), widget=TransInput(),
+                            required=False)
+    title = TransField(label=_lazy(u'Title'), widget=TransInput(),
+                       required=False)
+    url = forms.URLField(label=_lazy(u'URL'))
```

Comment:
If it's definitely required then yeah, we probably want to not allow it to be empty at the model level too. Should we do that here or in a follow up commit?